src/components/lib/jitutils/doc.md

```diff
@@ -0,0 +1,7 @@
+## jitutils
+
+### Description
+
+This library provides utilities for JIT compilation of mpk callgates.
```

Comment:
@evanstella @gparmer  From here I think maybe we can set the lib name to be `mpk_jitutils`?